src/editor.c

```diff
@@ -1404,20 +1559,34 @@ static gint get_xml_indent(ScintillaObject *sci, gint line)
 }
 
 
-static gint get_indent_size_after_line(GeanyEditor *editor, gint line)
+/* returns whether the indentation *level* changed.
+ * new sizes and alignment are returned in @p size_ and @p align */
+static gboolean get_indent_size_after_line(GeanyEditor *editor, gint line,
+		gint *size_, gint *align, gint *line_matched)
```

Comment:
trailing underscore seem a relatively common practice for output parameters, but yeah it could probably get a better name